.github/workflows/integration-tests.yml

```diff
@@ -121,6 +124,7 @@ jobs:
         continue-on-error: true
     outputs:
       src: ${{ steps.ignore-filter.outputs.changes || steps.changes.outputs.changes }}
+      ccip-changes: ${{ steps.ignore-filter.outputs.changes || steps.changes.outputs.ccip-changes }}
```

Comment:
@kalverra I think you need to use this output in other jobs/steps

Reply:
[It is here](https://github.com/smartcontractkit/chainlink/blob/e41d0e485aa0a2c3dc6acf909bc5c09da0fc80f7/.github/workflows/integration-tests.yml#L524)

Reply:
ah, right!